test/git_environment.go

```diff
@@ -292,6 +268,40 @@ func (env GitEnvironment) TagTable() (result DataTable, err error) {
 	return builder.Table(), nil
 }
 
+func (env GitEnvironment) initializeWorkspace(repo *GitRepository) error {
+	return repo.Shell.RunMany([][]string{
+		{"git", "config", "git-town.main-branch-name", "main"},
+		{"git", "config", "git-town.perennial-branch-names", ""},
+		{"git", "checkout", "main"},
+		// NOTE: the developer repos receives the master branch from origin
+		//       but we don't want it here because it isn't used in tests.
+		{"git", "branch", "-d", "master"},
+	})
+}
+
+func (env GitEnvironment) setupMemoizedRepository(repo *GitRepository) error {
+	// Since we copied the files from the memoized directory,
+	// we have to set the "origin" remote to the copied origin repo here.
+	_, err := repo.Shell.Run("git", "remote", "remove", "origin")
```

Comment:
We shouldn't have to remove the origin here since the memoized repo doesn't have one.

Reply:
I changed the creation of the memoized repo to keep the origin and to remove it here instead (so I could reuse the helper for the setup of the coworker environment.